browser_use/browser/session.py

```diff
@@ -975,7 +967,7 @@ async def setup_new_browser_context(self) -> None:
 			else:
 				raise
 
-		if self.browser_profile.stealth and not isinstance(self.playwright, Patchright):
```

Comment:
can you put this back, we only need the warning if the user passed in a non-patchright obj

Reply:
sure